portfolio/src/main/webapp/index.html

```diff
@@ -7,12 +7,39 @@
     <script src="script.js"></script>
   </head>
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
+    <div class="header">
+      <h1 class="page-heading">
+        Karol Portfolio
+      </h1>
+      <div class="social">
+        <a href="http://github.com/karol22"  target="_blank" >
+          <img  class="logo-media" src="images/logos/github.png" />
+        </a>
+        <a href="https://www.linkedin.com/in/karol-jose-gutierrez-suarez-06a445132/" target="_blank">
+          <img class="logo-media" src="images/logos/linkedin.png" />
+        </a>
+      </div>
+    </div>
+    <div id="content" class="main center">
+      <img class="profile-picture" src="images/selfie.png" />
+      <h2 class="greeting">Hi, I'm Karol Gutiérrez</h2>
+      <p>
+        I am a senior Computer Science student from Mexico. I study at Tecnológico
+        de Monterrey in Guadalajara.
+        My main interests are programming and mathematics. I love to build things 
+        from scratch and learn new things. I want to develop something that helps 
+        many people as well as bring technology to the youth of Mexico.
+      </p>
+
+      <br/>
+      <button class="center" onclick="addRandomQuote()">Click here to get a random quote from Avatar: The Last Airbender</button>
+      <p id="quote-container">
+        "The past can be a great teacher." – Aang
+      </p>
+      <br/>
+      <button class="center" onclick="changeCat()">Click here to get a random image of some of my cats:</button>
+      <br/>
+      <img id="cat-image" class="center" src="images/cats/cat_1.jpeg" />
```

Comment:
When a user hovers over the image, because this is the last item on the page, it extends the page beyond what was previously visible, so part of the picture is obscured.

Would it make sense to add some pixel/margin-buffer below this item or at the bottom of this item, so you can scroll to the bottom before, and the picture is then not obscured when enlarged?